### ISSUE TITLE
Mapillary "Street object ****" is - still - responsible for too many false positives.

### ISSUE THREAD

**Issue by AnkEric** [osm-fr/osmose-backend#524]
Street object **** observed around but not associated tags.
Mapillary "Street object ****" is - still - responsible for too many false positives. Even after removing the "street object = bench".

@frodrigo: _For street objects the main part is about missing traffic light_

My suggestion:
Restrict Osmose Errors (Warnings) to "street object = traffic light - upright"

"Traffic Light" (too many) false positives:

- "traffic light - single" (might be just a light (lamp) above a directional sign)
- "traffic light - horizontal" (most often: warning railway crossing, already tagged as railway crossing)
- "traffic light - other" (false positive or less relevant)

"Surveillance Camera" (too many) false positives:

- ventilation channel output (mounted in the wall)
- light (lamp) (mounted on the wall)
- motion detector (mounted on the wall)

Also (IMO): Me Myself and I don't like Surveillance Cameras to be mapped on OSM.
I know for a fact (some) governments, administrators, private house owners agree.
Burglars would love to see surveillance cameras on OSM.
In some countries having Speed Cameras on a GPS is an offence. 
So I (personally) don't map Surveillance Cameras and I would prefer this not to be considered an error.

"Bicycle Parking" (too many) false positives:

- barrier=fence
- barrier=cycle_barrier
- barrier=gard_rail

Using Mapillary "Street object ****" for OSM Mapping is useful. Very useful!
Even for "benches" (False Positive Champion and rightly removed as Osmose Error).
But marking ALL missing associated tags on Osmose as an Error is exaggerated (IMO) and ignores priorities.

But... also... deleting ALL (except for "traffic light - upright"), like I suggest, has its drawbacks.
For instance: "traffic light - horizontal" might also indicate a bridge=movable, which is too often missing on OSM.
_Don't throw the baby out with the bathwater...!_

**Comment by zzptichka** [comment]
FWIW in some countries/states/provinces all traffic lights are horizontal. So definitely need to keep those.

**Comment by AnkEric** [comment]
FWIW:

> But... also... deleting ALL (except for "traffic light - upright"), like I suggest, has its drawbacks.
> For instance: "traffic light - horizontal" might also indicate a bridge=movable, which is too often missing on OSM.
> Don't throw the baby out with the bathwater...!

Also:
It might be an idea to contact Mapillary about these false positives (a wooden gate is not a bench, a "ventilation channel output" is not a traffic light).
I don't know if anyone from OSM has already contacted Mapillary. In the Netherlands both Osmose and Mapillary is used insufficiently by OSM. Oké: IMO and just my estimate.
Might also be a suggestion: a "traffic light" should have more then one light. One light is more likely to be a false positive... (warning lights only) + ????
Oké, I will ask Mapillary (I was mostly complaining about my own images -))

**Comment by zzptichka** [comment]
Yeah you can help mapillary train their AI: https://help.mapillary.com/hc/en-us/articles/115001964171-Verifier-tool

**Comment by zzptichka** [comment]
As for lights at railway crossings, it's not false positive. Make sure your crossings have `crossing:light=yes` tag. It was discussed here: https://github.com/osm-fr/osmose-backend/issues/380#issuecomment-488733886

Also, for adding things like benches and bicycle parking based on Mapillary I highly recommend using Pic4Review. It's better suited for this kind of job. See this mission for example: https://pic4review.pavie.info/#/mission/666

**Comment by AnkEric** [comment]
Response from Mapillary:

_Hi Eric, thanks for the email!

Regarding the surveillance cameras, I'm not sure about the legal status of detecting them right now. Aside from that, we've been in contact with the maintainer of Osmose about these items, and he has already been removing some in the past due to false positives. We encourage people to set up an organization on Mapillary.com/app and draw a shape, then do a data verification task for their area of interest, which will actually improve the data by removing false positives. (https://help.mapillary.com/hc/en-us/articles/360025532692-Verification-projects). We are working on more solutions to directly receive false positive reports from apps like Osmose, and make the verification tool more present to users who are consuming the data for OSM.

We have received a good amount of feedback from OSM communities, including France, Belgium, Japan, Australia, USA, and Canada about these detections and the problems with them. Our priority now is again to encourage verification projects by the community to improve the data. We're also integrated with Pic4Review which uses our data, and Pic4Review allows skipping Mapillary data if it is a false positive, and provides us direct feedback of which unique point data were skipped. We plan to make this more advanced.

I hope this helps, let me know if you have any other questions and have a great day!_
